packages/core/src/components/Tooltip/index.tsx

```diff
@@ -185,6 +185,11 @@ export class Tooltip extends React.Component<Props & WithStylesProps, State> {
     const { cx, styles, theme, width: widthProp, content, inverted } = this.props;
     const { open, targetRect, tooltipHeight } = this.state;
 
+    // render null until targetRect is initialized by cDM
+    if (targetRect.top === 0 && targetRect.left === 0) {
+      return null;
+    }
+
```

Comment:
I added this to `renderPopUp` versus in the render at 

```jsx
{open && !(targetRect.top === 0 && targetRect.left ===0) ? this.renderPopUp() : null}
```
because it's more of a concern of the PopUp rendering and less about the business logic knowing it's open. If we extracted the PopUp into its own component, this is where the logic should live.